com.github.jerempw.gnonograms.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "com.github.jeremypw.gnonograms",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.jeremypw.gnonograms",
+
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
```

Comment:
does this work under wayland? using fallback-x11 without wayland access is... unusual.

Reply:
OK, I thought that arg implied Wayland.  I havent actually tried it under Wayland but it should work.